src/main/java/core/basesyntax/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReportGenerator;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public String generateReport(Map<String, Integer> inventory) {
```

Comment:
```suggestion
    public String generateReport() {
```
Don't pass storage as a parameter. You can call it directly